src/main/java/org/jboss/xavier/analytics/pojo/input/workload/inventory/VMWorkloadInventoryModel.java

```diff
@@ -36,6 +36,8 @@
     private String version;
     private String host_name;
     private boolean cpuAffinityNotNull;
+    private boolean numaNodeAffinityNotNull;
```

Comment:
In the MA-OCPv validation rules spreadsheet it is said that numaNodeAffinity is a String field

Reply:
I agree, though looking at the validation rules spreadsheet cpu affinity is defined as an array of int or null - will the integration project convert this into Boolean?

Reply:
just remembered that bit had already been done

Reply:
for numaNodeAffinity are we just checking that it is not null to raise the flag? There's no mention of requiring a particular value.

Reply:
In fact , I don't see any mention to raise a flag. ¿?

Reply:
The JIRA comments mention a flag foe each of the 3 fields

Reply:
VM/Host affinity/anti-affinity - generate Flag if has_vm_affinity_config equals true
CPU affinity                         - generate Flag if cpu_affinity not null
NUMA node affinity              - generate Flag if numa_node_affinity not null

Reply:
rules altered to match this spec